probe-rs/src/probe/daplink/mod.rs

```diff
@@ -148,7 +165,7 @@ impl DAPLink {
         if self.batch.is_empty() {
             return Ok(0);
         }
-        log::debug!("Processing batch of {} items", self.batch.len());
+        debug!("Processing batch of {} items", self.batch.len());
```

Comment:
I think using `log::debug!` is more concise here. I generally try to use all the log statements with `log::`.

Reply:
Sure thing. There's only bare `debug!()` in this file except for these two, would you like me to change all of them to use `log::debug!` or just revert the change to these ones?

Reply:
Let's leave it for now and do a cleanup commit later on :)